.chloggen/feat_issue-3127_support-image-pull-secrets-in-crds.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: crds
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Support for Configuring imagePullSecrets for Collector and Agents with CRDs
```

Comment:
Support for configuring imagePullSecrets for collector and auto-instrumentation images